src/index.ts

```diff
@@ -59,7 +59,7 @@ async function run(): Promise<void> {
       body && (params.body = body)
       discussionCategoryName && (params.discussion_category_name = discussionCategoryName)
 
-      void octokit.rest.repos
+      await octokit.rest.repos
```

Comment:
Take a look carefully.

It doesn't needed to be awaited since no necessary code after the code.

Reply:
Ok. I will fix it.